### ISSUE TITLE
Engineering mode move energy not showing anymore

### ISSUE THREAD

**Issue by benturnbull1991** [madmicio/lovelace-ld-2410-control-card#9]
Great project!
I haven't been able to get the live move energy graph since a previous version. I noticed that `gmoveenergie` was defined as `gmoveenergie1` in src. Could this be what is causing the issue?

https://github.com/madmicio/lovelace-ld-2410-control-card/blame/e5752f4f709c31d084dbc5ac2808e7d950f705e4/src/lovelace-ld2410-control-card.ts#L209

**Comment by madmicio** [comment]
yes this is the problem.  I had already noticed the error but was waiting for hacs default to approve a new release.  but times are getting longer.  I will make the correction as soon as possible.  the card works correctly with our firmware.  among the advantages there is also a significant reduction in the number of entities created.
